src/main/java/tempest/GUI/AddSessionPage.java

```diff
@@ -0,0 +1,208 @@
+package tempest.GUI;
+
+import tempest.Module;
+import tempest.State;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.time.Duration;
+import java.util.Date;
+import java.util.Objects;
+
+//TODO:
+// - Add a check so the user can't enter study sessions in a day which add up to be > 24hrs
```

Comment:
This has been fixed.

Reply:
I don't think it has - I'm working on this atm. It's different to a single study session being greater than 24 hours (what Josh did earlier)